### ISSUE TITLE
Rename `imageName` into `image`

### ISSUE THREAD

**Issue by gbartolini** [cloudnative-pg/cloudnative-pg#1089]
`imageName` should be simply called `image`

**Comment by xoxys** [comment]
I think I was wrong with the helm-manager and renaming it to `image` should just work with the renovate Kubernetes manager.

**Comment by DavidSpek** [comment]
You can use the regex manager in Renovate instead so you don’t need to rename anything. https://docs.renovatebot.com/modules/manager/regex/

**Comment by xoxys** [comment]
You are right, that's what I am using at the moment https://github.com/thegeeklab/renovate-presets/blob/main/kubernetes.json#L37-L45

However, it might still be handy to make it compatible with an existing manager.

**Comment by phisco** [comment]
just a note: it would be the [kubernetes manager](https://docs.renovatebot.com/modules/manager/kubernetes/) that would automatically detect the image if we switched to `image` (if enabled) in any Cluster resource.

**Comment by vishalanarase** [comment]
👋 Hi @gbartolini, I'd like to work on this!

Please assign this issue to me if it’s available — happy to get started with your guidance!
